enterprise/templatetags/enterprise.py

```diff
@@ -0,0 +1,30 @@
+"""
+Template tags for enterprise apps.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django import template
+from django.utils.safestring import mark_safe
+
+
+register = template.Library()  # pylint: disable=invalid-name
+
+
+@register.simple_tag(name="fa_icon")
```

Comment:
This might be better as an [inclusion tag](https://docs.djangoproject.com/en/1.11/howto/custom-template-tags/#inclusion-tags).